hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/heartbeat/HoodieHeartbeatClient.java

```diff
@@ -262,6 +262,7 @@ private void updateHeartbeat(String instantTime) throws HoodieHeartbeatException
       heartbeat.setLastHeartbeatTime(newHeartbeatTime);
       heartbeat.setNumHeartbeats(heartbeat.getNumHeartbeats() + 1);
     } catch (IOException io) {
+      LOG.error("Unable to generate heartbeat,instant:{}", instantTime, io);
       throw new HoodieHeartbeatException("Unable to generate heartbeat ", io);
     }
```

Comment:
Strength the error msg should be fine:

```java
throw new HoodieHeartbeatException("Unable to generate heartbeat for instant " + instantTime, io);
```

Reply:
We can remove the log: `LOG.error("Unable to generate heartbeat,instant:{}", instantTime, io);`